tests/functional/spec/features/supplementary_data/supplementary_data.spec.js

```diff
@@ -320,6 +323,14 @@ describe("Using supplementary data", () => {
     ]);
     await assertSummaryValues(["100 kg", "200 kg", "50 kg", "300 kg", "£110.00", "£220.00", "£330.00"]);
     await click(Section6Page.submit());
+    await expect(await $(HubPage.summaryRowState("section-6")).getText()).toBe("Completed");
+  });
+
+  it("Given I am using a supplementary dataset with 2 items in the product list, When I enter the production targets section, Then I only see an interstitial block as the question is skipped", async () => {
```

Comment:
Minor, and may be just me, but I think this (and maybe some of the other added tests in other files) should be more generic, so that the behaviour being tested is clear, rather than explicitly stating 2 items and the name of the list etc.

Reply:
Yeah I see what you mean, I've gone through and updated some of the descriptions to try and make them a bit more generic/specifc to whats actually being tested rather than the schema setup. If any others jump out let me know. I didn't take out some mentions of product list and section names because I thought if I went much further it might then be confusing when different lists are testing different things, hopefully this is slightly better